velox/exec/Task.cpp

```diff
@@ -2145,14 +2145,13 @@ std::string Task::toJsonString() const {
   }
 
   if (planFragment_.planNode) {
-    obj["plan"] = planFragment_.planNode->toString();
+    obj["plan"] = planFragment_.planNode->toString(true, true);
   }
 
   folly::dynamic driverObj = folly::dynamic::object;
   int index = 0;
   for (auto& driver : drivers_) {
-    driverObj[std::to_string(index++)] =
-        driver ? driver->toJsonString() : "null";
+    driverObj[std::to_string(index++)] = driver ? driver->toDynamic() : "null";
```

Comment:
Can you use folly::array here?

Reply:
Actually I think it will be nice to have an index:driver pair shown in the print other than a pure list of drivers without index. It will be more readable. WDYT?

Reply:
That's fine, but how would you use the index?